packages/body/tests/site/test.scss

```diff
@@ -0,0 +1,4 @@
+@import '../../../core/dist/sass/module.scss';
```

Comment:
probably we should alias the root so this would read:

`+@import 'core/dist/sass/module';`

instead